autoload/LanguageClient.vim

```diff
@@ -6,30 +6,36 @@ function! s:Echo(message) abort
     echo a:message
 endfunction
 
-function s:Ellipsis(message) abort
-    let l:maxlen = &columns * &cmdheight - 2
-    if &showcmd
-        let maxlen -= 11
-    endif
-    if &ruler
-        let maxlen -= 18
-    endif
-    if len(a:message) < l:maxlen
-        let l:message = a:message
-    else
-        let l:message = a:message[:l:maxlen - 3] . '...'
-    endif
-    return l:message
+" Credit: ALE, snippets from ale#cursor#TruncatedEcho()
+function! s:Truncate(message, cmd) abort
```

Comment:
Can you add also the replacing of line endings?

Reply:
I don't understand very well. What does `the replacing of line endings`? Anyway, I have pushed a commit to revert the `s:EchoEllipsis()` related changes. Correct me if anything is not suitable.

Reply:
I meant replacing line endings and tabs from original ALE implementation,

https://github.com/w0rp/ale/blob/9674132933df3e5b126c30da33a26c9be3ef87c6/autoload/ale/cursor.vim#L17

Reply:
Done.